src/components/Display.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+class Display extends React.Component { /* eslint-disable-line react/prefer-stateless-function */
```

Comment:
* I get this error when the page loads kindly fix it 🙏 
![image](https://user-images.githubusercontent.com/52497246/129308777-fbbbb02a-1a73-4f65-aa8a-ffe87fb21012.png)

Reply:
That's not really an error, it's a warning, and it is unrelated to the requirements. Most importantly, the `key` property is used mostly for `<li>` elements, of which there are **none** in this app. Furthermore, React uses `key` to keep track on elements in case they are removed from the DOM (or changed). Neither of those things will happen in a **Calculator** app.

Reply:
* Keys help React identify which items have changed, are added, or are removed. Keys should be given to the elements inside the array to give the elements a stable identity this is best practice and makes your code look professional 
* Hint 💭 the error is coming from `keyboard.js` where you have an array of props `keyValues` you can add a key prop to the `Key` component 🙏